dbscheme-proposal.md

```diff
@@ -0,0 +1,120 @@
+# Database Scheme
+
+## scheme
+* type (enc/sig)
+* name
+* category (lattice etc)
+* exact problems plus type of reduction (list of tuples?)
```

Comment:
Maybe this should go to another group (together with trust) that's called flavor or variant?

Reply:
further pull request incoming